src/circuit/range_chip.rs

```diff
@@ -38,58 +50,70 @@ impl<N: FieldExt> RangeChip<N> {
     }
 
     pub fn configure(meta: &mut ConstraintSystem<N>) -> RangeChipConfig {
+        assert!(VALUE_COLUMNS > 1);
+
         let block_first = meta.fixed_column();
         let range_class = meta.fixed_column();
         let tag_range_table_column = meta.lookup_table_column();
         let max_range_table_column = meta.lookup_table_column();
-        let value = meta.advice_column();
+        let values = [(); VALUE_COLUMNS].map(|_| meta.advice_column());
 
-        meta.enable_equality(value);
+        meta.enable_equality(values[0]);
```

Comment:
Because we expose the last col on assign_single_range, so the last column should also be enabled for quality. We need to rearrange these columns.